bundles/org.openhab.binding.openweathermap/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -67,6 +67,15 @@
 			<add-channel id="sunset" groupIds="forecastDaily">
 				<type>openweathermap:sunset</type>
 			</add-channel>
+			<add-channel id="moonrise" groupIds="forecastDaily">
```

Comment:
Create a new instruction set after the existing one and put these three instructions into it (please fix the formatting with `mvn spotless:apply -pl :org.openhab.binding.openweathermap`):

```xml
<instruction-set targetVersion="2">
    <add-channel id="moonrise" groupIds="forecastDaily">
        <type>openweathermap:moonrise</type>
    </add-channel>
			<add-channel id="moonset" groupIds="forecastDaily">
				<type>openweathermap:moonset</type>
			</add-channel>
			<add-channel id="moon_phase" groupIds="forecastDaily">
				<type>openweathermap:moon_phase</type>
			</add-channel>
</instruction-set>
```